cli.js

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env node
 'use strict';
+
+const spawn = require('child_process').spawn;
+
 const kexec = require('kexec');
```

Comment:
`kexec` fails when windows users install it. It should be replaced (I recommend `cross-spawn` like in my PR #22)